client/jetpack-onboarding/controller.js

```diff
@@ -20,3 +23,19 @@ export const onboarding = ( context, next ) => {
 	context.primary = <JetpackOnboardingMain stepName={ context.params.stepName } />;
 	next();
 };
+
+export const siteSelection = ( context, next ) => {
```

Comment:
Since the "other" `siteSelection` is so ubiquitously used, maybe we should rename this to something more specific, to reduce risk of confusing them.

Reply:
Hey, I named it this way only because I wanted you to favor this approach 😆

Reply:
But seriously, got any good naming ideas? Just `site` or `selectSite` or something?